janusgraph-hbase-parent/janusgraph-hbase-core/src/main/java/org/janusgraph/diskstorage/hbase/HBaseStoreManager.java

```diff
@@ -356,14 +358,17 @@ public HBaseStoreManager(org.janusgraph.diskstorage.configuration.Configuration
 
     @Override
     public Deployment getDeployment() {
-        List<KeyRange> local;
-        try {
-            local = getLocalKeyPartition();
-            return null != local && !local.isEmpty() ? Deployment.LOCAL : Deployment.REMOTE;
-        } catch (BackendException e) {
-            // propagating StorageException might be a better approach
-            throw new RuntimeException(e);
+        if (deployment == null) {
```

Comment:
In Java, this is only issue if `deployment` is a boolean, but I feel vaguely nostalgic for some of the craziness assignment in test can cause in C/C++